src/decisionengine/framework/engine/de_query_tool.py

```diff
@@ -6,21 +6,24 @@
 
 def create_parser():
     parser = argparse.ArgumentParser()
-    parser.add_argument("product", metavar="<product>", help="product to query")
-    parser.add_argument("--format", metavar="<format>", help="Possible formats are 'csv', 'json'.")
-    parser.add_argument(
+    optional = parser.add_argument_group("optional arguments")
+    optional.add_argument("--format", metavar="<format>", help="Possible formats are 'csv', 'json'.")
+    optional.add_argument(
         "--since",
         metavar="<time>",
         help="Minimum start time for task managers. "
         "If omitted, searches only the current task manager.\n"
         "(e.g. 2021-03-21 11:00:00)",
     )
-    parser.add_argument("--port", metavar="<port number>", default="8888", help="Default port is 8888")
-    parser.add_argument("--host", metavar="<hostname>", default="localhost", help="Default hostname is 'localhost'")
-    parser.add_argument(
+    optional.add_argument("--port", metavar="<port number>", default="8888", help="Default port is 8888")
+    optional.add_argument("--host", metavar="<hostname>", default="localhost", help="Default hostname is 'localhost'")
+    optional.add_argument(
         "-v", "--verbose", action="store_true", help="Include exception message in printout if server is inaccessible"
     )
 
+    positional = parser.add_argument_group("positional arguments")
+    positional.add_argument("product", metavar="<product>", help="product to query")
+
```

Comment:
Similar to my previous comment on optional arguments. Arguments w/o dash are positional arguments, why making it explicit?